lib/Jsrt/ChakraDebug.h

```diff
@@ -538,7 +538,6 @@
             _In_ unsigned int objectHandle,
             _Out_ JsValueRef *handleObject);
 
-#ifdef _WIN32
```

Comment:
`ch` references this method whether it is used or not. Option 1) disable from CH for nonWindows. Option 2) assert for non-Windows.